### PR TITLE
Simplify text of tooltips

### PR DESCRIPTION
**Pull request by damofer**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [x] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number:

### CODE REVIEW

**Comment by damofer** [comment]
Just simplify text for something shorter

File: src/messages.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-  "pagination.prev": "Go to previous page",
-  "pagination.next": "Go to next page",
-  "pin.toolbar": "Pin the controls toolbar",
-  "fullscreen.on": "Go fullscreen",
+  "pagination.prev": "Preview",
```
Comment (btzr-io) [inline src/messages.json]: Should be: `Previous page`

File: src/messages.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  "pagination.next": "Next",
```
Comment (btzr-io) [inline src/messages.json]: `Next page`

File: src/messages.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+  "pin.toolbar": "Pin controls",
+  "fullscreen.on": "Fullscreen",
   "fullscreen.off": "Exit fullscreen",
-  "view.bookmode": "View in book mode (double page view)",
-  "view.singlepage": "View as a single page",
-  "theme.light": "Switch to light theme",
-  "theme.dark": "Switch to dark theme",
+  "view.bookmode": "Book mode",
```
Comment (btzr-io) [inline src/messages.json]: Change to: Book view

File: src/messages.json [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+  "view.singlepage": "Single page",
```
Comment (btzr-io) [inline src/messages.json]: Page view

File: src/messages.json (hunk 1 above)
Comment (damofer) [reply src/messages.json]: Got it

File: src/messages.json (hunk 2 above)
Comment (damofer) [reply src/messages.json]: Got it

File: src/messages.json (hunk 3 above)
Comment (damofer) [reply src/messages.json]: Got it

File: src/messages.json (hunk 4 above)
Comment (damofer) [reply src/messages.json]: Got it

File: src/messages.json [hunk 5]
```diff
@@ -1,13 +1,13 @@
 {
-  "pagination.prev": "Go to previous page",
-  "pagination.next": "Go to next page",
-  "pin.toolbar": "Pin the controls toolbar",
-  "fullscreen.on": "Go fullscreen",
+  "pagination.prev": "Preview",
+  "pagination.next": "Next",
+  "pin.toolbar": "Pin controls",
```
[the remark this answers is not in the document]
Comment (btzr-io) [reply src/messages.json]: I forgot about this one ^^

**Pull request merged**